### PR TITLE
review my changes 

### PR DESCRIPTION
**Pull request by ibrahimmm92**
I have written my scenarious and made some changes for all of us

### CODE REVIEW

File: src/test/java/com/nextbase/pages/BasePage.java [hunk 1]
```diff
@@ -32,13 +32,13 @@ public void navigateToModule(String module){
...
-//
-//        sendMessageField.click();
-//        WebElement btn = Driver.getDriver().findElement(By.xpath("//button[contains(@id,'blog-submit-button')][.='" + button + "']"));
-//        btn.click();
-//
-//    }
+    public void sendAndCancelBtn(String button){
+    //Send or Cancel button
+        sendMessageField.click();
+        WebElement btn = Driver.getDriver().findElement(By.xpath("//button[contains(@id,'blog-submit-button')][.='" + button + "']"));
```
Comment (AinuraOrozobaeva) [inline src/test/java/com/nextbase/pages/BasePage.java:38]: why you use findElement() method instead of @FindBy method?

Comment (ibrahimmm92) [reply src/test/java/com/nextbase/pages/BasePage.java:38]: Because it is dynamic method

**Review by AinuraOrozobaeva — APPROVED** [review]
I created one more feature files just for your reference, you can delete after

**Review by ibrahimmm92 — COMMENTED** [review]
Thank you :)

**Pull request merged**